### ISSUE TITLE
Active Effect prototypeToken.sight.range does not affect token sight range.

### ISSUE THREAD

**Issue by SDoehren** [foundryvtt/dnd5e#4705]
Active Effect prototypeToken.sight.range does not affect sight range.

As seen below the Active Effect prototypeToken.sight.range adding 60 changes the sight range listed in the console but does not change the range of the token itself.

![image](https://github.com/user-attachments/assets/e9ee8a81-61f3-46f2-84e6-d87bc4fd41be)

**Comment by JPMeehan** [comment]
Yes, this is related to how the core software handles active effects.
